src/dune_engine/dune_project.ml

```diff
@@ -668,9 +662,28 @@ let parse ~dir ~lang ~opam_packages ~file =
      and+ wrapped_executables =
        field_o_b "wrapped_executables"
          ~check:(Dune_lang.Syntax.since Stanza.syntax (1, 11))
-     and+ allow_approx_merlin =
-       field_o_b "allow_approximate_merlin"
-         ~check:(Dune_lang.Syntax.since Stanza.syntax (1, 9))
+     and+ _allow_approx_merlin =
```

Comment:
I was about to suggest to remove it from the docs but it wasn't even documented in the manual. But there's a hint to add it somewhere so I suppose the code to produce is now dead and should be removed if that's not the case.

Reply:
The current behaviour is:

- If dune_lang < 2.7 this option is a no-op
- If dune_lang >= 2.8 it raises a warning that it is a no-op

The code raising an hint to use that option has already been deleted.